src/frontend/components/PrivateRoute/PrivateRoute.js

```diff
@@ -1,15 +1,14 @@
-import React from "react";
 import { useSelector } from "react-redux";
-import { Navigate, useLocation } from "react-router-dom";
+import { Navigate, Outlet, useLocation } from "react-router-dom";
 
-const PrivateRoute = ({ children }) => {
-  let location = useLocation();
-  const { isAuth } = useSelector((store) => store.auth);
+const PrivateRoute = () => {
+  const { isAuth } = useSelector((state) => state.auth);
+  const location = useLocation();
 
   return isAuth ? (
-    children
+    <Outlet />
   ) : (
-    <Navigate replace to="/signin" state={{ from: location }} />
+    <Navigate to="/signin" state={{ from: location }} replace />
```

Comment:
Can you please verify this replace functionality once. I can able to see sign in page when we click back

![image](https://user-images.githubusercontent.com/72279128/177099537-4381a849-d55a-4af6-8dd1-06aa25688f2e.png)

Reply:
fixed